assets/style.css

```diff
@@ -0,0 +1,266 @@
+body {
+    margin: 0;
+    padding: 0;
+    font-family: monospace;
+}
+
+nav {
+    display: flex;
+    align-items: flex-start;
+    width: 100%;
+    height: 56px;
+    background: white;
+    position: fixed;
+    top: 0px;
+    box-shadow: 1px 1px 1px 1px rgb(240, 240, 240);
+    z-index: 1;
+}
+
+nav > a .apps {
+    position: absolute;
```

Comment:
Please try your best to have DRY(don't repeat yourself) code.
line 20, 28, 36, 42, 49, 56, 62, 69 could be written in one liner.
example; 
```
nav, a, span {
  position: absolute;
}
```